Cargo.toml

```
@@ -20,6 +20,9 @@ version = "0.3.8"
 
 [workspace.dependencies]
 bytes = { version = "1.7" }
+futures-core = { version = "0.3" }
```

is it better to use local path in dev environment?
```suggestion
futures-core = { version = "0.3", path = "./fusio-core" }
```

---

But this is futures-core, not fusio-core

---

I got mistake